src/applications/vaos/services/location/transformers.js

```diff
@@ -157,7 +157,7 @@ function transformOperatingHours(facilityHours) {
  * @returns A facility id with either 442 or 552 replaced with 983 or 984
  */
 function getTestFacilityId(facilityId) {
-  if ((!environment.isProduction() && facilityId) || window.Cypress) {
+  if (facilityId && (!environment.isProduction() || window.Cypress)) {
```

Comment:
Minor tweak to this conditional to ensure there is a facility id passed to this function if we are running Cypress tests